frontends/web/src/routes/settings/components/mobile-header.module.css

```diff
@@ -38,7 +38,7 @@
 }
 
 .withGuide .headerText {
-    left: calc(50% + 37px);
+    left: calc(50% + 22px);
```

Comment:
22 is the width of the back btn without the margin (which is invisible). This is to visually centre the header text.